### ISSUE TITLE
Feature Request: Consistency in generated webfont character IDs.

### ISSUE THREAD

**Issue by phyllisstein** [Templarian/MaterialDesign#253]
Hey there! I hate to be a nudzh, especially w/r/t a project of which I'm so completely enamored---seriously, thank you for all your hard work, it's a thing of absolute beauty---but I was wondering if there was any way to adjust the webfont build process so that the character IDs stayed more or less consistent across builds.

It's probably a bit of an edge case (though hopefully not an _extreme_ edge case), but in at least one project with which I'm using the webfont, I have CSS that looks like this:

```css
.members-banner__member--featured::after {
  bottom: 0;
  color: #ffeb3b;
  content: '\f431';
  font-family: 'MaterialDesignIcons';
  font-size: 0.791em;
  pointer-events: none;
  position: absolute;
  right: 0;
}
```

As of MDI 0.9.72-alpha, that added a pretty little star to any element that got the class. But every time the build changes, I have to go back and make sure it's still a pretty little star, rather than a pretty little cloud or (my personal favorite) pretty little Clippy. Is there any way to, say, have the build task sort the SVGs by date added, rather than by alphabetizing them?

Thanks in advance for considering it! And best of luck with your continued development.

**Comment by Templarian** [comment]
Sorry, this has been considered, but is most likely not going to be fixed. Eventually the site will allow the generation of subsets of fonts that will help solve this issue a little bit. The plan is to build the icon set so large that most will use subsets of the fonts.

Ideally if you are using SASS to compile your CSS you can include the `*.scss` which contains two lists (`$hexes` and `$names`). The indexes map to one another, so you could write a simple helper function.

```scss
@function nameToChar($name) {
    @return content: char(nth($hexes, index($names, $name)));
}
```
(note char is defined in `_icons.scss`).

The designer font that is being built will use fixed character codes. In issue #247, but this will only include the `.ttf`.

**Comment by phyllisstein** [comment]
Gotcha. I'm using Stylus, but may have to consider switching over---thanks for the hugely helpful code-snippet, and the speedy response!

**Comment by Templarian** [comment]
I'm focusing on adding LESS next, but in the future I'll look into Stylus support (no one has requested support for it, or even mentioned it so I've been ignoring it for now).

**Comment by Templarian** [comment]
This was resolved in the v1.5.54 release if you are not following the main thread.

**Comment by phyllisstein** [comment]
That's great! Thanks so much for the heads-up!
